src/DevBetterWeb.Core/Interfaces/IPaymentHandlerSubscription.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace DevBetterWeb.Core.Interfaces
+{
+  public interface IPaymentHandlerSubscription
+  {
+    DateTime GetStartDate(string subscriptionId);
+    DateTime GetEndDate(string subscriptionId);
```

Comment:
Will these frequently be called one after the other by whatever client code uses this interface? Is there a way to combine them into one method that returns a date range type or Tuple?